Player.py

```diff
@@ -25,13 +26,15 @@ def __init__(self, x, y, playerId, screen):
         self.fullMoveTimeframe = constants.FPS * self.PLAYER_ANIMATION_SPEED_MULTIPLIER
         self.bombsHandler = BombsHandler()
         self.map = MapClient(self.screen)
+        self.newBoard = self.map.board
         self.shouldPlayerMove = True
 
+
     def draw(self):
         self.updatePlayerAnimationState()
         image = self.getPlayerImage()
         self.screen.blit(image, (self.x, self.y))
-        self.bombsHandler.drawBombs(self.screen)
+        self.newBoard = self.bombsHandler.drawBombs(self.screen, self.map.board)
 
```

Comment:
holy 💣 I know that you're still developing this, but why didn't you use `updateBoard` method in the first place?